pkg/controllers/dynakube/otelc/config.go

```diff
@@ -1,9 +1,9 @@
-package otel
+package otelc
 
 import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
 )
 
 var (
-	log = logd.Get().WithName("extension-otel")
+	log = logd.Get().WithName("otelc")
```

Comment:
wdyt about renaming to otel-collector? or do you think it is too long?

Reply:
it's good idea will do :)

Reply:
fixed in 1e7389728